apps/test/unit/applab/applabTest.js

```diff
@@ -623,4 +623,16 @@ describe('applab', () => {
       expect(howItWorksIndex).to.be.below(reportAbuseIndex);
     });
   });
+
+  describe('Applab.getHtmlForWidgetMode', () => {
+    it('changes the width of screens', () => {
+      // Applab project with two screens, a label, a button, and an image.
+      Applab.levelHtml = `<div id="designModeViz" class="appModern clip-content" tabindex="1" data-radium="true" style="width: 320px; height: 450px;"><div class="screen" tabindex="1" data-theme="default" id="screen1" style="display: block; height: 450px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0; background-color: rgb(255, 255, 255);"><button id="button1" style="padding: 0px; margin: 0px; border-style: solid; height: 40px; width: 100px; background-color: rgb(255, 164, 0); color: rgb(255, 255, 255); border-color: rgb(77, 87, 95); border-radius: 4px; border-width: 1px; font-family: &quot;Arial Black&quot;, Gadget, sans-serif; font-size: 15px; position: absolute; left: 85px; top: 40px;">Button</button><label style="margin: 0px; line-height: 1; overflow: hidden; overflow-wrap: break-word; max-width: 320px; border-style: solid; text-rendering: optimizespeed; color: rgb(77, 87, 95); background-color: rgba(0, 0, 0, 0); border-color: rgb(77, 87, 95); border-radius: 0px; border-width: 0px; font-family: &quot;Arial Black&quot;, Gadget, sans-serif; font-size: 13px; padding: 2px 15px; width: 60px; height: 18px; position: absolute; left: 110px; top: 110px;" id="label1">text</label></div><div class="screen" tabindex="1" data-theme="default" id="screen2" style="display: none; height: 450px; width: 320px; left: 0px; top: 0px; position: absolute; z-index: 0; background-color: rgb(255, 255, 255);"><img src="/blockly/media/1x1.gif" data-canonical-image-url="" data-image-type="" data-object-fit="contain" id="image1" style="height: 100px; width: 100px; border-style: solid; border-width: 0px; border-color: rgb(0, 0, 0); border-radius: 0px; position: absolute; left: 105px; top: 140px; margin: 0px;"></div></div>`;
```

Comment:
this is definitely a nit, but this HTML is ~1,700 characters long -- can we use a simpler example here? it doesn't need to be technically exact, just enough to get the point across and test any edge cases

Reply:
hmm, I kind of think it's better to have the test case be a real example, since the use-case is using this function to copy the exact html text. And the way design mode/level html works is that all of the styles are included inline on each element, so it does get long, but I think it's important to cover with a test case that the *only* thing that changes is the width of the screen elements. I don't feel that strongly though, so lmk if you want me to do a fake example instead